js/ermrest.js

```diff
@@ -62,14 +62,18 @@ var ERMrest = (function (module) {
      * URI should be to the ERMrest _service_. For example,
      * `https://www.example.org/ermrest`.
      */
-    function getServer(uri) {
+    function getServer(uri, params) {
 
         if (uri === undefined || uri === null)
             throw new module.InvalidInputError("URI undefined or null");
 
+        if (typeof params === 'undefined' || params === null) {
+            params = {'cid': null};
```

Comment:
it's okay if `params` is `null`. we do not need to initialize it to `{'cid': null}`. just leave it null and downsteam you just won't add the params to the url.